lib/settings_page/settings_page_widget.dart

```diff
@@ -21,15 +21,6 @@ class _SettingsPageWidgetState extends State<SettingsPageWidget> {
 
   List<Widget> _buildSpotifyForm() {
     return [
-      TextFormField(
-        controller: _spotifyIDController,
-        decoration:
-            const InputDecoration(labelText: 'Enter Your Client ID Here:'),
-      ),
-      TextFormField(
-          controller: _spotifyClientSectetController,
-          decoration: const InputDecoration(
-              labelText: 'Enter Your Client Secret Here:')),
```

Comment:
What happened to these? Do we no longer need them? If so is the config page even needed?

Reply:
See my comment on the above question.

I would like to still keep the settings page in case we introduce other music providers that you can choose from.